packages/terra-demographics-banner/src/selectable-demographics-banner/SelectableDemographicsBanner.module.scss

```diff
@@ -0,0 +1,68 @@
+:local {
+  .selectable-demographics-banner {
+    background-clip: border-box;
+    background-color: var(--terra-demographics-banner-selectable-background-color, transparent);
+    background-image: var(--terra-demographics-banner-selectable-background-image);
+    box-shadow: var(--terra-demographics-banner-selectable-box-shadow);
+    color: var(--terra-demographics-banner-selectable-color, #fff);
+    line-height: 1;
+    padding: var(--terra-demographics-banner-selectable-padding, 0.714em);
+  }
+
+  .profile-photo {
+    margin: var(--terra-demographics-banner-selectable-profile-photo-margin);
+  }
+
+  .background-tile {
+    background-color: #004c76;
+    display: flex;
+    flex-direction: row;
+  }
+
+  .deceased {
+    background-clip: border-box;
+    background-color: var(--terra-demographics-banner-deceased-selectable-background-color, #1c1f21);
+    background-image: var(--terra-demographics-banner-deceased-selectable-background-image);
+    color: var(--terra-demographics-banner-deceased-selectable-color, #fff);
+
+    .profile-photo {
+      opacity: var(--terra-demographics-banner-deceased-selectable-profile-photo-opacity, 0.7);
+    }
+  }
+
+  .selectable-tile {
+    border: solid 1px transparent;
+    border-radius: 4px;
+    box-shadow: 0 0 3px 1px rgba(0, 0, 0, 0.33);
+    flex: 1 1 auto;
+    margin: 0.5rem;
+
+    &:focus {
+      border-color: #c8cacb;
+    }
+
+    &:hover {
+      border-color: #c8cacb;
+    }
+  }
+
+  .divider {
+    border: 1px solid #fff;
+    border-radius: 1.5rem;
+    flex: 0 0 auto;
+    margin: 0.75rem 0.1rem;
```

Comment:
i notice the divider is shorter than the selectable tile - should we reduce the top and bottom divder margins to better match them?
![Screen Shot 2020-03-11 at 9 27 18 AM](https://user-images.githubusercontent.com/14063284/76427651-94184580-637a-11ea-823d-a3f39d374386.png)

Reply:
I think divider is shorter than selectable tile in [zeplin callouts](https://app.zeplin.io/project/5a8c5310e635140e3cc134c2/screen/5e3c0c23dcada891b1bd737b) as well 

![Uploading Screen Shot 2020-03-16 at 1.18.23 PM.png…]()